### DISCUSSION TITLE
Linking large file folder with the repository

### DISCUSSION THREAD

**Discussion by alexschellong** [community/community#40232] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Is it somehow possible to "link" a big (111GB) folder with an identical Existing Large File storage repository?  (or is deleting it and cloning the repository the only way.)

The files have been transferred to me via a hard drive, but now I would like to add it to GitHub desktop for source control reasons. :  - ) 

Thank you !

**Comment by ossanna16** [comment]
Hi there @alexschellong and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
